src/routes/oath.ts

```diff
@@ -162,9 +163,11 @@ router.get('/oauth-callback', async (req, res) => {
             const get_user_id = await axios.get('https://api.hubapi.com/oauth/v1/refresh-tokens/'+resp.data.refresh_token,get_options);
 			//#todo temporarily using user email for tenant alias rather than id
 			// as the db does not support number tenant alias currently
-			hubspotID = get_user_id.data.user;
+			current_user = get_user_id.data.hub_id;
+
 			// #todo in a seperate ticket check first whether the user already exists in DB
-            AddTokensToDatabase(hubspotID, resp.data.access_token, resp.data.refresh_token)
+            AddTokensToDatabase(current_user.toString(),resp.data.access_token, resp.data.refresh_token)
+
 			// store user id in local tokenStore for knowledge of current user
 			// and for knowing which user to poll the DB
             tokenStore["userID"] = hubspotID;
```

Comment:
Did we still need this token store? Or can we remove it now?

Reply:
I have a ticket up for refactoring OAuth I'll remove that and other dead code in this ticket.